WordPressVIPMinimum/Tests/Classes/RestrictedClassesUnitTest.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class OkTestClass extends WPCOM_VIP_CLI_Command { } // Ok.
```

Comment:
Add messages _why_ these are OK, and _why_ the ones below are not (and how the test cases differ) - especially relevant if there are later other classes added to this RestrictedClasses sniff.